src/ServicePulse.Host/vue/src/components/EventLogItem.vue

```diff
@@ -0,0 +1,186 @@
+<script setup>
+import { useRouter } from "vue-router";
+import TimeSince from "../components/TimeSince.vue";
+
+defineProps({
+  eventLogItem: Object,
+});
+const router = useRouter();
+
+function navigateToEvent(eventLogItem) {
+  switch (eventLogItem.category) {
+    case "Endpoints":
+      router.push("/configuration/endpoint-connection");
```

Comment:
these routes are essentially defined elsewhere (in index.js), so should be exported constants from a module

Reply:
Switched to named routes (which still uses magic strings but it's better). Given the relatively low number of pages, I wonder if we should have a `navigate` service or something `navigate.toEndpointConnection();`

Reply:
@mikeminutillo from previous experience, I would still make it composable, but make the links an exported object.
Example:
```js
const routeLinks = {
  dashboard: {
     root: "/",
  },
  configuration: {
     root: "/configuration",
     connection: "/configuration/connections",
  },
  ....
}

export default routeLinks;
```
and then:
```js
import routeLinks from "routeLinks"

<a href="routeLinks.configuration.root">Configuration</a>
```